File: Exemplos M10/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_ReactNative-font/src/intentDigitalHubService/BRIDGE/SetServer.ts
import {BridgeCommand} from './BridgeCommand';

export class SetServer extends BridgeCommand {
  readonly ipTerminal: string;
  readonly portaTransacao: number;
  readonly portaStatus: number;

  constructor(ipTerminal: string, portaTransacao: number, portaStatus: number) {
    super('SetServer');
    this.ipTerminal = ipTerminal;
    this.portaTransacao = portaTransacao;
    this.portaStatus = portaStatus;
  }

  functionParameters(): object {
    return {
      ipTerminal: this.ipTerminal,
      portaTransacao: this.portaTransacao,
      portaStatus: this.portaStatus,
    };
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Bridge/SetServer.cs
namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Bridge
{
    class SetServer : BridgeCommand
    {
        readonly string ipTerminal;
        readonly int portaTransacao;
        readonly int portaStatus;

        public SetServer(string ipTerminal, int portaTransacao, int portaStatus) : base("SetServer")
        {
            this.ipTerminal = ipTerminal;
            this.portaTransacao = portaTransacao;
            this.portaStatus = portaStatus;
        }

        protected override string FunctionParameters()
        {
            return "\"ipTerminal\"" + ":" + "\"" + ipTerminal + "\"" + "," +
                    "\"portaTransacao\"" + ":" + portaTransacao + "," +
                    "\"portaStatus\"" + ":" + portaStatus;
        }
    }
}